vendor/bat-native-ads/src/bat/ads/internal/catalog/campaign/creative_set/catalog_creative_set_info.h

```diff
@@ -42,6 +42,7 @@ struct CatalogCreativeSetInfo final {
   unsigned int total_max = 0;
   double value = 0.0;
   std::string split_test_group;
+  std::vector<float> embedding;
```

Comment:
`embeddings`

Reply:
See other usages of embedding where it is plural, thanks

Reply:
each ad is represented by a single vector, which constitutes one embedding